inference-engine/src/offline_transformations/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+# Copyright (C) 2018-2019 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET_NAME "offline_transformations")
```

Comment:
`moc_transformations`?

Reply:
decided in offline to have offline_transformations